app/assets/stylesheets/components.scss

```diff
@@ -194,6 +194,22 @@ section .year-expand {
   }
 }
 
+// Search results page _______________________________
+
+// Pagination
+.paginate{
```

Comment:
Note: we should start doing BEM, otherwise it'll be a lot of work to convert everything later.